quiche/src/recovery/gcongestion/recovery.rs

```diff
@@ -706,10 +707,15 @@ impl RecoveryOps for GRecovery {
         if let Some(thresh) = spurious_pkt_thresh {
             self.pkt_thresh =
                 self.pkt_thresh.max(thresh.min(MAX_PACKET_THRESHOLD));
+            self.time_thresh = PACKET_REORDER_TIME_THRESHOLD;
         }
 
         if self.newly_acked.is_empty() {
-            return Ok(OnAckReceivedOutcome::default());
+            return Ok(OnAckReceivedOutcome {
+                acked_bytes,
+                spurious_losses,
+                ..Default::default()
+            });
```

Comment:
Found a bug while testing. Without this we dont report the spurious count in the following case:


```
Ack packet: 1
[0, 1, 2, 3, 4, 5]
      ^

Ack packet: 0           // this is spurious loss, but since we didn't ack new packets we return 0 because of `default()`.
[0, 1, 2, 3, 4, 5]
 ^  x


```